docs/usage/release-notes-for-major-versions.md

```diff
@@ -7,6 +7,49 @@ The most recent versions are always at the top of the page.
 This is because recent versions may revert changes made in an older version.
 You also don't have to scroll to the bottom of the page to find the latest release notes.
 
+## Version 38
+
+### Breaking changes for 38
+
+General:
+
+- Require Node.js 20 ([#30291](https://github.com/renovatebot/renovate/pull/30291))
+- The Renovate Docker images no longer have `-slim` tags. You must stop using the `-slim` prefix. Renovate now defaults to the `-slim` tag type behavior.
+
+Specific:
+
+- **bitbucket-server:** autodetect `gitAuthor`, if possible ([#29525](https://github.com/renovatebot/renovate/pull/29525))
+- **config:** change from `boolean` to `enum` for `onboardingNoDeps`. Renovate now onboards repositories with no dependencies, with one exception: if you run Renovate in `autodiscover` mode then you must manually onboard Renovate for repos with no dependencies
+- **config:** sanitize special characters from branch names for vulnerability type PRs. This may cause Renovate to autoclose/replace existing PRs
+- **config:** change the order of `globalExtends` resolution, it is applied _first_ and remaining global config takes precedence
+- **datasource/docker:** Docker Hub lookups prefers `hub.docker.com` over `index.docker.io`. To revert to the old behavior: set `RENOVATE_X_DOCKER_HUB_TAGS_DISABLE=true` in your env
+- **git:** check _all_ commits on the branch to decide if the branch was modified ([#28225](https://github.com/renovatebot/renovate/pull/28225))
+- **gitea:** use "bearer auth" instead of "token auth" to authenticate to the Gitea platform
+- **github:** if you run Renovate as a GitHub app then `platformCommit` is automatically enabled
+- **http:** remove `dnsCache`
+- **logging:** you must set file logging via env, not in `config.js`
+- **manager/pep621:** change `depName` for `pep621` dependencies. This also changes the branch name for `pep621` updates, which may cause Renovate to autoclose and re-open some `pep621` PRs
+- **npm:** for npm versions lower than 7, drop support for remediating vulnerabilities in _transitive_ dependencies
+- **npm:** remove `RENOVATE_CACHE_NPM_MINUTES` ([#28715](https://github.com/renovatebot/renovate/pull/28715))
+- **packageRules:** `matchPackageNames` (and related functions) no longer fall back to checking `depName`
+- **packageRules:** `matchPackageNames` exact matches are now case-insensitive
+
+### Commentary for 38
+
+For maintainers to fill out.
+You should probably explain:
+
+- The Node.js 20 change
```

Comment:
Quote from `@rarkins`:

> Chosen because of https://nodejs.org/en/blog/vulnerability/july-2024-security-releases

And quote from `@viceice` about "why we you not mandating just the minimum version of Node 20?":

> yes, and we decided to limit supported versions.

- #30376

Reply:
we dropped node v18 support and decided to require a non-vulnerable node v20 version

Reply:
- When another Node.js 20 security update is released, will we update the `engines` field again?
- If we update the `engines` field, is that a `major`, `minor` or `patch` type release for the Renovate package?